src/storage/secondary/manifest.rs

```diff
@@ -164,7 +166,12 @@ impl SecondaryStorage {
             return Err(TracedStorageError::duplicated("table", table_name));
         }
         let table_id = schema
-            .add_table(table_name.clone(), column_descs.to_vec(), false)
+            .add_table(
+                table_name.clone(),
+                column_descs.to_vec(),
+                false,
+                ordered_pk_ids,
```

Comment:
`ordered_pk_ids` also needs to be persisted to disk in manifest.rs `CreateTableEntry`. We can do this in later PRs.